ank/src/cli_commands/dry_run_plan.rs

```diff
@@ -0,0 +1,123 @@
+use std::collections::{BTreeMap, BTreeSet};
+use tabled::Tabled;
+use super::cli_table::CliTable;
+use common::objects::WorkloadInstanceName;
+
+#[derive(Clone, Debug)]
+pub enum DryRunAction {
+    NoChange,
+    Create,
+    Recreate,
+    Delete,
+}
+
+impl std::fmt::Display for DryRunAction {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let s = match self {
+            DryRunAction::NoChange => "no change",
+            DryRunAction::Create   => "create",
+            DryRunAction::Recreate => "recreate",
+            DryRunAction::Delete   => "delete",
```

Comment:
Changing to the new action strings:
```suggestion
            DryRunAction::Add => "added",
            DryRunAction::Update => "updated",
            DryRunAction::Delete => "deleted",
```